graspy/embed/oosase.py

```diff
@@ -0,0 +1,416 @@
+# oosase.py
+# Created by Hayden Helm on 2019-04-22.
+# Email: hhelm2@jhu.edu
+import warnings
+
+from .base import BaseEmbed
+from .svd import selectSVD
+from ..utils import import_graph, get_lcc, is_fully_connected, is_symmetric
+
+import numpy as np
+from sklearn.utils.validation import check_is_fitted
+from sklearn.utils import check_array
+import networkx as nx
+
+
+class OutOfSampleAdjacencySpectralEmbed(BaseEmbed):
+    r"""
+    Class for computing the out of sample adjacency spectral embedding of a graph.
+    
+    The adjacency spectral embedding (ASE) is a k-dimensional Euclidean representation of 
```

Comment:
make all lines <88 characters. This won't be an issue in most of the code but black can't fix docstrings